subsys/net/lib/lwm2m/lwm2m_obj_firmware.c

```diff
@@ -335,8 +334,8 @@ static struct lwm2m_engine_obj_inst *firmware_create(uint16_t obj_inst_id)
 			  &update_result, sizeof(update_result));
 	INIT_OBJ_RES_OPTDATA(FIRMWARE_PACKAGE_NAME_ID, res, i, res_inst, j);
 	INIT_OBJ_RES_OPTDATA(FIRMWARE_PACKAGE_VERSION_ID, res, i, res_inst, j);
-	INIT_OBJ_RES_DATA(FIRMWARE_UPDATE_PROTO_SUPPORT_ID, res, i, res_inst, j,
-			  &supported_protocol, sizeof(supported_protocol));
+	INIT_OBJ_RES_MULTI_OPTDATA(FIRMWARE_UPDATE_PROTO_SUPPORT_ID, res, i,
+			  res_inst, j, 1, false);
```

Comment:
Not a big deal, but it still isn't aligned
```suggestion
	INIT_OBJ_RES_MULTI_OPTDATA(FIRMWARE_UPDATE_PROTO_SUPPORT_ID, res, i,
				   res_inst, j, 1, false);
```
```suggestion
	INIT_OBJ_RES_MULTI_OPTDATA(FIRMWARE_UPDATE_PROTO_SUPPORT_ID, res, i,
			  res_inst, j, 1, false);
```